dependencies.yaml

```diff
@@ -553,15 +561,30 @@ dependencies:
     specific:
       - output_types: [requirements]
         matrices:
-          - matrix: {cuda: "12.*"}
+          - matrix:
+              cuda: "12.*"
             packages:
               - --extra-index-url=https://download.pytorch.org/whl/cu121
+          - matrix:
+              cuda: "13.*"
+            packages:
+              # TODO: switch from nightlies to releases when there are pytorch CUDA 13 released packages
+              #  ref: https://github.com/pytorch/pytorch/issues/159779
+              - --extra-index-url=https://download.pytorch.org/whl/nightly/cu130
```

Comment:
@alexbarghi-nv I want to draw your attention to these `torch` changes. Proposing, for now:

* using `torch` nightlies for CUDA 13.0
* proposing a floor of `torch >= 2.9.0a0` for CUDA 13 tests

This seems to be working ok. And hopefully we could switch away from nightlies as soon as there is a release with CUDA 13 support.

Reply:
This is fine with me.